core/app-framework/README.md

```diff
@@ -1,28 +1,27 @@
-Application framework
-=======
-
-## Directory structure
+# Application framework
 
+By using the iwasm VM core, we are flexible to build different application frameworks for the specific domains, although it would take quite some effort.
```

Comment:
```suggestion
By using the WAMR VM core, we are flexible to build different application frameworks for the specific domains, although it would take quite some effort.
```